test/integ/test_gluon_hosting.py

```diff
@@ -13,25 +13,26 @@
 from __future__ import absolute_import
 
 import json
+import os
 
-import pytest
+from sagemaker.mxnet.model import MXNetModel
 
-import docker_utils
-import utils
+import local_mode
 
 
 # The image should support serving Gluon-created models.
-@pytest.mark.skip
 def test_gluon_hosting(docker_image, opt_ml, processor):
```

Comment:
Does the mnist test use gluon? If not should we add a training test for gluon?

Reply:
it doesn't. adding a training test with gluon is probably a good idea - I've noted it in the things I have left to do, and will address it in a later PR (time permitting)